SabberStoneCoreTest/src/CardSets/LootapaloozaCardsGenTest.cs

```diff
@@ -3330,23 +3331,19 @@ public void RinTheFirstDisciple_LOOT_415()
 			game.StartGame();
 			game.Player1.BaseMana = 10;
 			game.Player2.BaseMana = 10;
-			//var testCard = Generic.DrawCard(game.CurrentPlayer, Cards.FromName("Rin, the First Disciple"));
-			game.Process(PlayCardTask.Any(game.CurrentPlayer, "Rin, the First Disciple"));
-
-			game.ProcessCard("Cataclysm", null, true);
```

Comment:
I think cataclysm discard could trigger other minions in play, like Silverware Golem.  messing up test state.